coriolis/tests/migrations/test_api.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2024 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from coriolis.migrations import api as migrations_module
+from coriolis.tests import test_base
+
+
+class APITestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis Migrations API."""
+
+    def setUp(self):
+        super(APITestCase, self).setUp()
+        self.api = migrations_module.API()
+        self.rpc_client = mock.MagicMock()
+        self.api._rpc_client = self.rpc_client
+        self.ctxt = mock.sentinel.ctxt
+        self.migration_id = mock.sentinel.migration_id
+
+    def test_migrate_instances(self):
+        origin_endpoint_id = mock.sentinel.origin_endpoint_id
+        destination_endpoint_id = mock.sentinel.destination_endpoint_id
+        origin_minion_pool_id = mock.sentinel.origin_minion_pool_id
+        destination_minion_pool_id = mock.sentinel.destination_minion_pool_id
+        instance_osmorphing_minion_pool_mappings = (
+            mock.sentinel.instance_osmorphing_minion_pool_mappings)
+        source_environment = mock.sentinel.source_environment
+        destination_environment = mock.sentinel.destination_environment
+        instances = mock.sentinel.instances
+        network_map = mock.sentinel.network_map
+        storage_mappings = mock.sentinel.storage_mappings
+        replication_count = mock.sentinel.replication_count
+        shutdown_instances = mock.sentinel.shutdown_instances
+
+        result = self.api.migrate_instances(
+            self.ctxt, origin_endpoint_id, destination_endpoint_id,
+            origin_minion_pool_id, destination_minion_pool_id,
+            instance_osmorphing_minion_pool_mappings, source_environment,
+            destination_environment, instances, network_map, storage_mappings,
+            replication_count, shutdown_instances, notes=None,
```

Comment:
Do not explicitly give the `kwargs` (`notes=None, skip_os_morphing=False, user_scripts=None`) when calling the tested function.

This way, we can cover with the unit test if those default values change, the assertion below will fail.

Reply:
Gotcha! You are right!